app/controllers/application_controller.rb

```diff
@@ -3,8 +3,17 @@ class ApplicationController < ActionController::Base
 
   rescue_from Pundit::NotAuthorizedError, with: :render_not_authorized
 
+  before_action :check_maintenance
   before_action :set_paper_trail_whodunnit
 
+  def check_maintenance
+    if FeatureToggle.maintenance_mode_enabled? && !%w[service-unavailable accessibility-statement privacy-notice cookies].include?(request.fullpath.split("?")[0].delete("/"))
```

Comment:
The exclusion of the `service-unavailable` path was to stop the app getting stuck in a loop (it wouldn't load the page without this).